README.md

```diff
@@ -0,0 +1,12 @@
+# xcovfe
```

Comment:
Ideally could we make this [like this README?](https://github.com/Xcov19/mycovidconnect#mycovidconnect)